five.ts

```diff
@@ -0,0 +1,15 @@
+interface Profile {
```

Comment:
Interface has to start with I. In this case `IProfile`
```ts
interface IProfile {
// ...
}
```

Reply:
มีงี้ด้วยอ่อวะ !!!

Reply:
Following Microsoft's guideline, capital I must be avoided.
https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines